blueprints/data-solutions/vertex-mlops/notebooks.tf

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_notebooks_runtime" "runtime" {
+  for_each = var.notebooks
+  name     = each.key
+
+  project  = module.project.project_id
+  location = var.notebooks[each.key].region
+  access_config {
+    access_type   = "SINGLE_USER"
```

Comment:
Is the single user instance the suggested one? This would require handle instance access from Terraform rather than through IAM

Reply:
I'll add this en the TODO too